nautobot/core/templates/generic/object_edit.html

```diff
@@ -11,11 +11,11 @@
         <div class="row">
```

Comment:
Might it make sense to change the default value for `settings.BRANDING_URLS.docs` (which is the URL linked by the "Docs" icon in the Nautobot page footer) to point to the static documentation files instead of `https://nautobot.readthedocs.io`?

Reply:
I was torn on this, and how you support the static url args as default or a regular url. Since you suggested this change, I will make it happen.